docs/proposals/backup/backup.md

```diff
@@ -0,0 +1,735 @@
+
+	// Schedule defines when to run the Backup using a Cron expression.
+	// A cron expression is a format used to specify the execution time of recurring tasks, consisting of multiple fields representing different time units.
+	// ┌───────────── minute (0 - 59)
+	// │ ┌───────────── hour (0 - 23)
+	// │ │ ┌───────────── day of the month (1 - 31)
+	// │ │ │ ┌───────────── month (1 - 12)
+	// │ │ │ │ ┌───────────── day of the week (0 - 6) (Sunday to Saturday;
+	// │ │ │ │ │                                   7 is also Sunday on some systems)
+	// │ │ │ │ │
+	// │ │ │ │ │
+	// * * * * *
+	// For example, "30 * * * *" represents execution at the 30th minute of every hour, and "10 10,14 * * *" represents execution at 10:10 AM and 2:10 PM every day.
+	// If not set, the backup will be executed only once.
+	// +optional
+	Schedule string `json:"schedule,omitempty"`
+
+	// Destination indicates the default clusters where backups should be performed.
+	// +optional
+	Destination *Destination `json:"destination,omitempty"`
+
+	// Policy are the rules defining how backups should be performed.
+	// +optional
+	Policy *BackupPolicy `json:"policy,omitempty"`
+}
+
+// Note: partly copied from https://github.com/vmware-tanzu/velero/pkg/apis/backup_types.go
+// BackupSpec defines the specification for a backup.
+type BackupPolicy struct {
+	// ResourceFilter specifies which resources should be included in the backup.
+	// It acts as a selective criterion to determine which resources are relevant for backup.
+	// If not set, the backup process will consider all resources. This filter helps in optimizing the backup process by excluding unnecessary data.
+	// +optional
+	ResourceFilter *ResourceFilter `json:"resourceFilter,omitempty"`
+
+	// TODO: consider SnapshotVolumes for backup
+
+	// TTL is a time.Duration-parseable string describing how long the Backup should be retained for.
+	// +optional
+	TTL metav1.Duration `json:"ttl,omitempty"`
+
+	// OrderedResources specifies the backup order of resources of specific Kind.
+	// The map key is the resource name and value is a list of object names separated by commas.
+	// Each resource name has format "namespace/objectname".  For cluster resources, simply use "objectname".
+	// For example, if you have a specific order for pods, such as "pod1, pod2, pod3" with all belonging to the "ns1" namespace,
+	// and a specific order for persistentvolumes, such as "pv4, pv8", you can use the orderedResources field in YAML format as shown below:
+	// ```yaml
+	// orderedResources:
+	//  pods: "ns1/pod1, ns1/pod2, ns1/pod3"
+	//  persistentvolumes: "pv4, pv8"
+	// ```
+	// +optional
+	// +nullable
+	OrderedResources map[string]string `json:"orderedResources,omitempty"`
+
+	// ItemOperationTimeout specifies the time used to wait for asynchronous BackupItemAction operations.
+	// The default value is 1 hour.
+	// +optional
+	ItemOperationTimeout metav1.Duration `json:"itemOperationTimeout,omitempty"`
```

Comment:
s/ItemOperationTimeout/Timeout

It is under backup, so we can simplify calling timeout